resources/testdata/glyphs3/WghtVar_AxisLocation.glyphs

```diff
@@ -15,7 +15,7 @@ date = "2022-12-01 04:52:20 +0000";
 familyName = WghtVar;
 fontMaster = (
 {
-axesValues = (400);
+axesValues = (0);
```

Comment:
https://github.com/googlefonts/fontmake-rs/issues/22#issuecomment-1352297087 says "design locations = xxxValue and user locations = "Axis Location" parameter" and I thought in Glyphs v3 we use axesValues instead of the v2 xxxValue?

Reply:
Is axesValues just not used if you have Axis Location then?

Reply:
Glyphs v3 axesValues are the masters' design locations, and "Axis Location" custom params are the respective user-space locations

Reply:
That's what I've done here isn't it? axesValues 0 because I want the design location to be 0 and Axis Location 400 because I want the user location to be 400?

Reply:
Yeah that's correct, but initially you had them inverted, and that's what I hinted at in the initial comment above.

Reply:
Got it! Sorry, I was fixated on the end result and missed the squash part